### DISCUSSION TITLE
Can it be used with this API?

### DISCUSSION THREAD

**Discussion by suphachok** [highfestiva/finplot#134] [Q&A]
Can anyone answer me if finplot is compatible with this? And how to use it, thank you
https://github.com/iqoptionapi/iqoptionapi
https://iqoptionapi.github.io/iqoptionapi/en/candle/candle/

**Comment by highfestiva** [comment]
From the looks of it, you should be able to do something like:

```python
...
data = Iq.get_realtime_candles(...)
df = pd.DataFrame(data).T
fplt.candlestick_ochl(df[['open','close','max','min']])
fplt.show()
```

**Reply by suphachok** [reply]
How do I get candlestick update realtime? Thank you

**Reply by highfestiva** [reply]
1. Learn to program.
2. Learn a bit of Python.
3. Research IQ Option API and/or read documentation.
4. Cobble together some code.

**Reply by suphachok** [reply]
Thank you sir ^^
